asydata/chimeraData/R3BAsyChimeraHitData.h

```diff
@@ -0,0 +1,45 @@
+#ifndef R3BAsyChimeraHitData_H
+#define R3BAsyChimeraHitData_H
+#include "TObject.h"
+
+class R3BAsyChimeraHitData : public TObject
+{
+
+  public:
+    /** Default constructor **/
+    R3BAsyChimeraHitData();
+
+    /** Constructor with arguments
+     *@param fPaddleId  Paddle number
+     *@param fX         Position X in [mm]
+     *@param fY         Position Y in [mm]
+     *@param fTof      Tof [ns]
+     **/
+    R3BAsyChimeraHitData(Int_t DetId, Double_t ELoss, Double_t Fast, Double_t Slow, Double_t ToF);
+
+    /** Destructor **/
+    virtual ~R3BAsyChimeraHitData() {}
+
+    /** Accessors **/
+    inline const Int_t& GetDetId() const { return fDetId; }
```

Comment:
With this line is enough:
Int_t GetDetId() const { return fDetId; }
remove the **inline const**

Reply:
in general